Etiquette/CODE_REVIEW.md

```diff
@@ -0,0 +1,110 @@
+* Use "we" instead of "you", unless asking for opinion of specific individual ("What do you think ...?", but "Here we are doing ... What if we ...?",)
+* Avoid using terms that could be seen as referring to personal traits. ("dumb",
+  "stupid", "insane"). Assume everyone is intelligent and well-meaning.
+* Be aware of negative bias with online communication. (If content is neutral, we assume the tone is negative.) Can you use positive language as opposed to neutral?
+* When using emojis be aware that their meaning can be different for different individuals - winking smile may not be always innocent
+* Same with any other media format you decide to use
+* Do not expect your comments to be answered immediately. If you need an answer fast - talk in person (the same applies to the Slack messages)
+* Accept that it's fine to have disagreements and it's hard to please everyone
+
+
+Creating a Pull Request
+--------
+
+* When creating your branches, name them using a format like `<author>/<ticket>` or `<author>/<ticket>-<shortdescription>`
+* Carefully fill in the pull request template:
+	* the title of the PR should follow the format `[XXX-123] Description of the change`, starting with a ticket number
+	* explain the context and motivation, don’t assume familiarity with the history. Highlight how the goal was achieved without going too much into details - people tend not to read long descriptions and if the long explanation is needed, then just a single pull request is probably not gonna be enough (i.e. the change may need to go through the proposal process or design review). 
+	* don't strip out parts of the template unless it is absolutely irrelevant to the changes (i.e. change in the build script does not need any screenshots as it does not touch UI)
+	* don't forget labels (note: it's better to set them after the PR is created, as GitHub sometimes fails with 405 status code and we don't know why)
+* Explicitly request review from or mention team mates you specifically want to involve in the discussion (in addition to the ones that will be assigned to it automatically by PullAssigners)
+* Self-review your changes, adding comments where you think you can get questions (it may worth adding these as code or documentation comments). Tip: if you spot small issues you can use GitHub suggestions and apply them right away
+* Keep your pull requests small, ideally they shouldn't be more than **800 additions/deletions**
+* Create a draft PR for work that is in progress, but only do that if you seek for opinions on your work
+* Mark PR with the "Blocked" label if it has any dependency that should stop it from being merged (it's fine for pull request be both "blocked" and "ready for review", but it shouldn't be "ready for review" and "wip")
+* Give your team mates a reasonable time to review your changes, if the change is critical use "Top priority" label and post it in the team channel
```

Comment:
```suggestion
* Give your team members a reasonable time to review your changes, if the change is critical use "Top priority" label and post it in the team channel
```

I know it's stretching it but especially in the uk `mate` is not really gender neutral (learned this the other day…). Better safe than sorry :)